Dockerfile

```diff
@@ -53,7 +66,7 @@ EXPOSE 8086
 
 VOLUME ["/data"]
 
-CMD ["/run.sh"]
+CMD ["sh", "-c", "/start.sh"]
```

Comment:
actually, "sh -c" is already the default entry point, the command should only be "/start.sh", else we would have sh -c sh -c start.sh